packages/eslint-config-4catalyzer/rules.js

```diff
@@ -80,4 +86,13 @@ module.exports = {
     },
   ],
   'import/prefer-default-export': 'off',
+  'import/order': [
```

Comment:
can we set `groups` to just `[['builtin', 'external']]`? or maybe `['builtin', 'external']` if you prefer?

Reply:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array